### PR TITLE
Jdoze/full wip

### PR DESCRIPTION
**Pull request by jasondoze**
Add deploy & destroy scripts, clean up README.md

### CODE REVIEW

**Review by ofelix60 — APPROVED** [review]
Looks great

File: README.md [hunk 1]
```diff
@@ -1,98 +1,89 @@
...
+- connect to the vm
 
-test the webserver
+- install the webserver
+
+- test the webserver
+
+&nbsp;
 
 ## **Windows**
```
Comment (stephenlauck) [inline README.md:28]: We should change the README.md to just showing how to run the appropriate script for your system:

Windows
`webserver.ps1`

Mac:
`zsh webserver.zsh`

...

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jasondoze) [reply README.md:28]: Changes requested added to README file.

**Review by stephenlauck — APPROVED** [review]
LGTM

**Pull request merged**